src/main/java/org/jabref/gui/StateManager.java

```diff
@@ -41,6 +45,17 @@
     private final OptionalObjectProperty<SearchQuery> activeSearchQuery = OptionalObjectProperty.empty();
     private final ObservableMap<BibDatabaseContext, IntegerProperty> searchResultMap = FXCollections.observableHashMap();
     private final OptionalObjectProperty<Node> focusOwner = OptionalObjectProperty.empty();
+    private final ObservableList<Task<?>> backgroundTasks = FXCollections.observableArrayList(taskProperty -> {
+        return new Observable[] {taskProperty.progressProperty(), taskProperty.runningProperty()};
+    });
+
+    public BooleanBinding anyTaskRunningBinding = Bindings.createBooleanBinding(
```

Comment:
Please make this fields private and add a public getter method (otherwise code from the outside can change these bindings, which is not what we want). (Maybe also remove the suffix "Binding" from the name, but that's a matter of taste).

Reply:
Sure thing!

Reply:
https://github.com/JabRef/jabref/pull/6443/commits/d56138bbb629dcd5ffdad9d29d9c7b87d80ba7c2